core/src/main/scala/design/hamu/Game.scala

```diff
@@ -0,0 +1,6 @@
+package design.hamu
+
+class Game() {
```

Comment:
I would suggest putting board here as a state of the game

and then maybe add a `update` method which returns a new `Game` with new`turn` and `isActive` and `board`

Reply:
```
class Game(val turn: Char, val isActive: Boolean, val board: Board) {
   def update(coordinate: String): Game = ...
}
```

Reply:
var game = // initial game state
```
while(game.isActive) {
    val coord = ...
    game = game.update(coord)
}
```